Decisions.Equifax/ConsumerCreditReport/Response/ConsumerCreditReportResponse.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Runtime.Serialization;
```

Comment:
This file cannot be changed much given existing customer usage. 
We need to test upgrade scenarios before merging this.

Reply:
In an earlier change, I changed it from Steps to Step, which does break the old step, reverted the change